utilities/ovs-vsctl.c

```diff
@@ -1020,7 +1042,13 @@ static struct cmd_show_table cmd_show_tables[] = {
      NULL,
      {&ovsrec_open_vswitch_col_manager_options,
       &ovsrec_open_vswitch_col_bridges,
-      &ovsrec_open_vswitch_col_ovs_version},
+      &ovsrec_open_vswitch_col_ovs_version,
```

Comment:
I just noticed that this change introduces a NULL following `&ovsrec_open_vswitch_col_ovs_version` when P4OVS is undefined, since it replaces `},` with `,},`. I don't know what effect this will have on the non-P4OVS code. (Not my edit)

Reply:
We can have #ifdef P4OVS and else to avoid any issues with non-P4OVS code